packages/manager/modules/manager-components/src/ssh-key/ssh-key.template.html

```diff
@@ -0,0 +1,51 @@
+<div>
+    <oui-field
+        data-ng-if="$ctrl.userSshKeys.length > 0 && !$ctrl.loading.init"
+        data-label="{{:: 'manager_components_select_ssh_key' | translate }}"
+        data-label-popover="{{:: 'manager_components_select_ssh_key_help' | translate }}"
+    >
+        <oui-select
+            id="sshKeys"
+            name="sshKeys"
+            data-items="$ctrl.userSshKeys"
+            data-match="keyName"
+            data-model="$ctrl.selectedSshKey"
+            data-on-change="$ctrl.getSshKeyInfo()"
+        ></oui-select>
+    </oui-field>
+
+    <div class="text-center" data-ng-if="$ctrl.loading.sshkey">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <div data-ng-if="!$ctrl.loading.sshkey">
+        <oui-field
+            data-label="{{:: 'manager_components_sshkey_input' | translate}}"
+        >
+            <div
+                class="float-right sshkey-switch"
+                data-ng-model="model.keyIsValid"
+                data-sshkey-switch
+                data-sshkey-switch-key="{{$ctrl.publicKey}}"
+            ></div>
+
+            <textarea
+                class="oui-input form-control"
+                name="sshKey"
+                id="sshKey"
+                placeholder="{{:: 'manager_components_sshkey_input_placeholder' | translate:{t0: $ctrl.ssh.inputRules.placeholder} }}"
+                data-ng-model="$ctrl.publicKey"
+                data-ng-change="$ctrl.selectedSshKey = ''"
+                data-ng-required="$ctrl.required"
+                data-ng-pattern="$ctrl.ssh.inputRules.pattern"
+                data-ng-trim="true"
+                spellcheck="false"
+                autocomplete="off"
+                autocapitalize="off"
+                autocorrect="off"
+                rows="{{ $ctrl.ssh.inputRules.rows }}"
```

Comment:
Can you make the rows value a parameter to that component ? (with a default value in case not provided)

Reply:
Same thing for the placeholder, can you make a parameter and if not provided, just a default value such as `ssh-rsa AAAAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX== my-public-key` without dependencies to a translation. So we pass the translated text to the component if we want to customize the text

Reply:
Otherwise for pattern, it's ok to keep it like you did :+1:

Reply:
Its done